src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public Rectangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg() {
+        return this.firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return this.secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
```

Comment:
The `getArea()` method is conceptually different from the `draw()` method. Consider separating them into different interfaces if they are not closely related in your project structure.